routingpy/routers/mapbox_osrm.py

```diff
@@ -412,9 +411,13 @@ def isochrones(
         if generalize:
             params["generalize"] = generalize
 
+        profile = profile.replace("mapbox/", "")
+
         return self._parse_isochrone_json(
-            self.client._request(
-                "/isochrone/v1/" + profile + "/" + locations_string, get_params=params, dry_run=dry_run
```

Comment:
we also changed the way we request, in order to be able to switch the client class for other purposes (we need that for other projects). you'll only have to change `self._request` to `self.client._request`.

Reply:
I was writing a suggest change, but you have to been to fast :)